### ISSUE TITLE
PR Harmony does not work with Reviewer Suggester plugin

### ISSUE THREAD

**Issue by rcdailey** [monitorjbl/pr-harmony#42]
Default reviewers are not filled into the reviewers field while creating a pull request if the Reviewer Suggester plugin is installed & enabled:
https://marketplace.atlassian.com/plugins/com.atlassian.stash.plugin.stash-suggest-reviewers/server/overview

Disabling the plugin results in PR Harmony working.

**Comment by monitorjbl** [comment]
Yeah, I recently did some investigation on another plugin that had issues with PR Harmony for #40. I strongly suspect this will wind up being the same problem. If other plugins want to independently write to the select box for reviewers on the Create PR screen, there's not a lot that can be done about it. It's not so much an incompatibility as it is two different people yelling over each other. One of the two is going to have to be the loudest and get the last word in. 

I could in theory make PR Harmony watch that select box and fill it back in with what it believes should be in there. However, that would simply move the burden onto the other plugin authors when they got cases complaining about their plugins not working with PR Harmony was installed.
